ui/components/app/snaps/snap-ui-renderer/index.scss

```diff
@@ -48,10 +48,6 @@
 
   &__panel {
     gap: 8px;
-
-    .mm-icon--size-inherit {
```

Comment:
I would imagine that this breaks this use-case: https://github.com/MetaMask/metamask-extension/pull/26904

Reply:
Let me check

Reply:
Fixed in bb3cc6e27be8f70cd0968c536848c0a2a0cab1a0

![Screenshot from 2024-10-21 17-03-34](https://github.com/user-attachments/assets/4851b69d-4e47-4fcf-9a87-d4ead6b72bff)